buildenv.sh

```diff
@@ -20,6 +20,68 @@
 
 set -o allexport
 
+PACKAGES=(
+  libbrotli-dev liblz4-dev libzstd-dev protobuf-compiler libgtest-dev
+  ffmpeg webp zipalign clang build-essential cmake openjdk-17-jdk
+  libncurses-dev bison flex libssl-dev libelf-dev python3
+  python-is-python3 cpio attr zip libprotobuf-dev make
+  libpcre2-dev ccache npm
+)
+
+echo "Checking dependencies..."
```

Comment:
This way you are considering that only Debian-based distros will be used for ROM building. You should adapt this logic to check if distro is Debian based. Possibly some packages are also available only in certain distros. 

Openjdk 11 is the minimum required version. And if I remember correctly also unzip package has to be installed.